test/terraform/test.tf

```diff
@@ -54,3 +54,31 @@ module "aws_deploy-test-ubuntu-bionic" {
     aws = "aws.ap-southeast-2"
   }
 }
+
+
+module "aws_deploy-test-ubuntu-focal" {
+  source              = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.3.1"
+  env                 = var.env_name
+  envid               = var.envid
+  bootstrap_version   = var.bootstrap_version
+  vault_role          = "ae-node"
+  vault_addr          = var.vault_addr
+  user_data_file      = "user_data.bash"
+  node_config         = "secret/aenode/config/test"
+
+  static_nodes = 1
+  spot_nodes   = 1
+
+  aeternity =  var.aeternity
+
+  spot_price    = "0.04"
+  instance_type = "t3.large"
+  ami_name      = "aeternity-ubuntu-20.04"
```

Comment:
let's run on stock ubuntu until https://github.com/aeternity/machine-image/pull/28 is merged

Reply:
@dincho it will not work, because we are looking only for images we own, https://github.com/aeternity/terraform-aws-aenode-deploy/blob/master/fleet/main.tf#L16 do you wish to copy this image, or add parameter for "owner" ?

Reply:
https://github.com/orgs/aeternity/projects/3#card-78171899